lua/ulx/modules/sh/fun.lua

```diff
@@ -994,14 +994,19 @@ end
 hook.Add( "PlayerDisconnected", "ULXMaulDisconnectedCheck", maulDisconnectedCheck, HOOK_MONITOR_HIGH )
 
 ------------------------------ Strip ------------------------------
-function ulx.stripweapons( calling_ply, target_plys )
+function ulx.stripweapons( calling_ply, target_plys, wep )
 	for i=1, #target_plys do
-		target_plys[ i ]:StripWeapons()
+		if wep and target_plys[ i ]:HasWeapon(wep) then
```

Comment:
I'm not sure if HasWeapon returns errors if the weapon class doesn't exists.
Maybe I'll test it tomorrow or later.